### ISSUE TITLE
[BUG] Shelly 2PM Roller (Cover) function not available in Homeassistant

### ISSUE THREAD

**Issue by DotNet2Web** [StyraHem/ShellyForHASS#684]
## Environment

- ShellyForHASS release with the issue: 1.0.0
- Last working ShellyForHASS release (if known): 
- Home Assistant Core release with the issue: Home Assistant Core 2022.6.2
- Operating environment (Home Assistant/Supervised/Docker/venv): Home Assistant Supervisor 2022.05.3

## Describe the bug

  Shelly 2 PM does not expose cover functionality

![image](https://user-images.githubusercontent.com/13330307/172045576-01f2444d-1819-4385-a408-ad0e04a81a02.png)

## Expected behavior

Expose the Cover (roller) functionality of the shelly 2PM, like the Shelly 2.5 already does.
Shelly 2.5: 
![image](https://user-images.githubusercontent.com/13330307/172045682-7220648e-5b48-4200-b7ae-1d1c07f036f8.png)

## Screenshots

![image](https://user-images.githubusercontent.com/13330307/172045591-f8de1826-7c90-4aaa-b073-a84fca46931d.png)

## Traceback/Error logs

```txt

```

**Comment by auanasgheps** [comment]
I have the same issue.

**Comment by d-edge82** [comment]
I have the same issue on 2pm, please help

**Comment by DotNet2Web** [comment]
Any news about this bug?

**Comment by gmantoha** [comment]
still need this bug fixed

**Comment by firstusing** [comment]
Same for me too. 
The Shelly plus 2 pm has been not seen as a Cover. 

Badly needed to see it as a Cover within HomeKit.

**Comment by cbakkerprive** [comment]
Hmm...
Experiencing exactly the same issue.
still no solution?

**Comment by Ulli2k** [comment]
some to me. Please fix it

**Comment by rinseringma** [comment]
Any updates on this issue?

**Comment by mbaluda-ms** [comment]
Same issue here

**Comment by Doom666DE** [comment]
Same issue

**Comment by lendy007** [comment]
I have the same issue...

**Comment by hansblom-com** [comment]
Any updates on this issue?

**Comment by hansblom-com** [comment]
Any updates on this issue?

**Comment by hansblom-com** [comment]
Any updates on this issue?

**Comment by Ringo982** [comment]
Any update?? Why this issue is still opened? Is the integration mantenance still active?

**Comment by hans-sein** [comment]
Please fix it.

**Comment by Thorvald-FR** [comment]
Need the solution too

**Comment by enryf89** [comment]
same problem here

**Comment by lendy007** [comment]
I was very annoyed by this problem so I went through painful process where I uninstalled Shelly2HASS and used native Shelly plugin available now in HA. I had to change entity ifs in most of my automations as their names are different from those generated by Shelly2Hass. But other than that, everything works like a charm and my 2PM is displayed correctly as cover.

**Comment by Ringo982** [comment]
Me tooI completely uninstalled shelly4hass integration and started with the native one.Now all it is fine and work properlyIt is not possible to wait for months for a change.. I suggest you to switch to the native integration because it is clear that shelly4hass is updated once every never….Inviato da iPhoneIl giorno 1 ott 2023, alle ore 16:22, lendy007 ***@***.***> ha scritto:﻿
I was very annoyed by this problem so I went through painful process where I uninstalled Shelly2HASS and used native Shelly plugin available now in HA. I had to change entity ifs in most of my automations as their names are different from those generated by Shelly2Hass. But other than that, everything works like a charm and my 2PM is displayed correctly as cover.
